packages/grafana-data/src/text/markdown.test.ts

```diff
@@ -8,11 +8,11 @@ describe('Markdown wrapper', () => {
 
   it('should sanitize by default', () => {
     const str = renderMarkdown('<script>alert()</script>');
-    expect(str).toBe('&lt;script&gt;alert()&lt;/script&gt;');
+    expect(str).toBe('');
```

Comment:
can DOMPurify keep this old behavior?  -- escaping the invalid HTML, but stripping invalid css properties?

Reply:
No, DOMPurify will remove content that is sanitized which is safer since escaped content might end up in other contexts, i.e. first sanitizing content, then run it through markdown which might convert the content back to arbitrary HTML.  

This might cause backwards compatibility problems, but looking at places where we already sanitize we do not rely on strings like `<foo>` to be interpreted which could be dangerous. 

We can add the `KEEP_CONTENT` option which will partly solve this, then `<foo>text</foo>` will become `text` instead of removing everything (`<script>alert()</script>` would still become ""). I recommend people to manually HTML escape their strings, like `&lt;foo&gt;text&lt;/foo&gt;` (will be seen as `<foo>text</foo>` in Text panel). From a risk perspective this is a better alternative then us trying to escape HTML as this would require more testing from our side.  

What do you think?